src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -156,6 +157,9 @@
 
   private static final Splitter DOT_SPLITTER = Splitter.on('.');
 
+  private static final ImmutableList<String> PACKAGES_TO_IGNORE =
+      ImmutableList.of("third_party/javascript/polymer/v2");
```

Comment:
Would be better for this to be a externally passed in flag. Also, needs more docs.